pkg/serialization/codec/jam/decode.go

```diff
@@ -50,6 +89,13 @@ func (br *byteReader) unmarshal(value reflect.Value) error {
 		return br.decodeFixedWidth(value, l)
 	case []byte:
 		return br.decodeBytes(value)
+	case BitSequence:
```

Comment:
We handle aliases in the `handleReflectTypes` we should follow same convention here
```
	case reflect.Slice:
		if value.Type() == reflect.TypeOf(ed25519.PublicKey{}) {
			return br.decodeEd25519PublicKey(value)
		} else if value.Type() == reflect.TypeOf(BitSequence{}) {
			return br.decodeBit(value)
		}
		return br.decodeSlice(value)

Reply:
Should this also have the byte slice in this case?

Reply:
yes you are right